keychain_unix.cpp

```diff
@@ -91,13 +99,18 @@ static KeyringBackend detectKeyringBackend()
     case DesktopEnv_Plasma5:
         return Backend_Kwallet5;
         break;
-        // fall through
+
+    // fall through
```

Comment:
(nitpick: fall through what?   there is already one return AND a redundent break :-) )

Reply:
Yes. I didn't add it.